Cargo.toml

```
@@ -38,6 +38,7 @@ semver = { version = "1.0.12", optional = true }
 sha1 = { version = "0.10.1", optional = true }
 sha2 = { version = "0.10.2", optional = true }
 sprintf = { version = "0.1.2", optional = true }
+parse-size = { version = "1.0.0", features=["std"], optional = true }
```

nitpick to keep the style consistent

```suggestion
parse-size = { version = "1.0.0", features = ["std"], optional = true }
```